web/packages/teleport/src/Roles/RoleEditor/StandardEditor.tsx

```diff
@@ -419,12 +437,166 @@ export function ServerAccessSpecSection({
   );
 }
 
-function KubernetesAccessSpecSection() {
-  // TODO(bl-nero): add the Kubernetes section
-  return null;
+export function KubernetesAccessSpecSection({
+  value,
+  isProcessing,
+  onChange,
+}: SectionProps<KubernetesAccessSpec>) {
+  return (
+    <>
+      <FieldSelectCreatable
+        isMulti
+        label="Groups"
+        isDisabled={isProcessing}
+        formatCreateLabel={label => `Group: ${label}`}
+        components={{
+          DropdownIndicator: null,
+        }}
+        value={value.groups}
+        onChange={groups => onChange?.({ ...value, groups })}
+      />
+
+      <Text typography="body3" mb={1}>
+        Labels
+      </Text>
+      <LabelsInput
+        disableBtns={isProcessing}
+        labels={value.labels}
+        setLabels={labels => onChange?.({ ...value, labels })}
+      />
+
+      <Flex flexDirection="column" gap={3} mt={3}>
+        {value.resources.map((resource, iRes) => (
+          <KubernetesResourceView
+            key={iRes}
+            value={resource}
+            isProcessing={isProcessing}
+            onChange={newRes =>
+              onChange?.({
+                ...value,
+                resources: value.resources.map((res, i) =>
+                  i === iRes ? newRes : res
+                ),
+              })
+            }
+            onRemove={() =>
+              onChange?.({
+                ...value,
+                resources: value.resources.toSpliced(iRes, 1),
```

Comment:
Oh nice, `toSpliced` is pretty cool, it might be the first time I see it. It caniuse.com says that [Chrome for Android got support for it just in September this year](https://caniuse.com/mdn-javascript_builtins_array_tospliced), but I guess it should be safe? I imagine that most of the time Chrome is updated separately from the OS.

I looked up `webassets/teleport/app/app.js` and it doesn't look like this gets compiled to a polyfilled version, it's just raw in there.

Reply:
@ravicious No, for some reason, caniuse.com only shows the last Android Chrome version (it's even [trickling down as a bug to other projects](https://github.com/browserslist/browserslist/issues/156#issuecomment-315416975)). It's been there since version 110, which was released in February 2023 (see [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSpliced#browser_compatibility)).

That being said, if mobile browsers are on our compatibility list, it should be documented [in the readme](https://github.com/gravitational/teleport/blob/eda3671595b6507c695dd460ae9d659f0505654a/web/README.md?plain=1#L177). Let me know if you have a more complete list of targets.

Reply:
That's good to hear regarding caniuse.com. I've just linked to it under another PR and was surprised to see Android Chrome again. Feb 2023 is good enough.

Regarding the target list, I'd like us to change it but I haven't had the time to advocate for that. It was set, in my opinion, [rather hastily](https://gravitational.slack.com/archives/C02DQ1C2BMW/p1657730403674659) (here's the PR it's been part of https://github.com/gravitational/webapps/pull/977). If you [plug it into `browsersl.ist`](https://browsersl.ist/#q=last+2+chrome+version%2C+last+2+edge+version%2C+last+2+firefox+version%2C+last+2+safari+version), you'll see that at the moment we "support" only 3.7% of global audience. Even if you filter that down to only North America, that's still just 4.8%. I think that's way too low, compared to 87.1% coverage that using the `defaults` target brings.